hack/development/kustomization.yaml

```diff
@@ -21,3 +21,4 @@ kind: Kustomization
 resources:
   - ../test
   - ../chains
+  - ../sigstore/fulcio
```

Comment:
Eventually, we want rekor and friends here as well, right?

Reply:
Is Rekor necessary for keyless?

Reply:
I _think_ it's possible to skip rekor, but it's definitely not something we want to do in a real environment. For instance, to disable rekor in cosign v2 you have to use the `--insecure-ignore-tlog` parameter. We can iterate here.

Reply:
Ended up using Rekor, need to see about ctlog and timestamp, perhaps in a subsequent PR...